plugins/scaffolder-backend/src/lib/files/serializeDirectoryContents.ts

```diff
@@ -62,19 +61,11 @@ export async function serializeDirectoryContents(
 
   const limiter = limiterFactory(10);
 
-  const valid = await asyncFilter(paths, async ({ dirent, path }) => {
+  const valid = await asyncFilter(paths, async ({ dirent }) => {
     if (dirent.isDirectory()) return false;
     if (!dirent.isSymbolicLink()) return true;
 
-    const safePath = resolveSafeChildPath(sourcePath, path);
-
-    // we only want files that don't exist
-    try {
-      await fs.stat(safePath);
-      return false;
```

Comment:
Thanks for the review @benjdlambert ✨ ! 

I'm not sure I fully understand your question ? What is the concern here ? 
I think that the scaffolder should treat the symlinks like any other files, it was surprising to us that it wasn't the case but even more that it was working for broken ones were but not valid ones

`git` itself treat symlinks but either resolving them or leaving them untouched, through the [core.symlinks global config](https://git-scm.com/docs/git-config)

Reply:
Hey :wave:

Maybe I'm misremembering, but I seem to believe that before there was behaviour that it used to resolve the when copying the files in `fetch:template` or something like that. I think treating them as symlinks rather than resolving is the way that we want to go, just want to make sure that we're not breaking behaviour if we're resolving them somewhere. @Rugvip can you remember what used to do the coping that was a problem? I can't remember what it was fully. Was it the `tempdir` and then copy to workspace?